django_socio_grpc/tests/grpc_test_utils/fake_grpc.py

```diff
@@ -323,6 +313,52 @@ async def __anext__(self):
         return response
 
 
+class FakeFullAioStreamUnaryCall(FakeFullAioCall, grpc.aio.StreamUnaryCall):
```

Comment:
is heriting from grpc.aio.StreamUnaryCall cannot rise issue about using function that call the linux socket instead of the FakeAsyncContext ?

Reply:
It's an abstract class, no functionality